src/components/ActionsPanel/components/ActionsList.tsx

```diff
@@ -70,20 +62,25 @@ export const ActionsList = ({ actionTemplates, onActionClick }: ActionsListProps
   );
 
   return (
-    <ActionsContainer>
-      {actions.map((action: ActionTemplate, i) => (
-        <ActionItem key={i} actionTemplate={action} onActionClick={onActionClick} />
-      ))}
+    <>
+      <Box maxHeight={{ md: '240px' }} overflowY="scroll">
```

Comment:
![image](https://user-images.githubusercontent.com/1561200/109951090-8bccbf00-7cdd-11eb-9e4b-29db43b7cf5d.png)
Broke dashboard layout

Reply:
Reduced size to align with token panel on full screen.